pl_examples/domain_templates/computer_vision_fine_tuning.py

```diff
@@ -271,25 +264,19 @@ def training_step(self, batch, batch_idx):
         num_correct = torch.eq(y_bin.view(-1), y_true.view(-1)).sum()
 
         # 3. Outputs:
-        tqdm_dict = {'train_loss': train_loss}
-        output = OrderedDict({'loss': train_loss,
-                              'num_correct': num_correct,
-                              'log': tqdm_dict,
-                              'progress_bar': tqdm_dict})
+        tqdm_dict = {"train_loss": train_loss}
+        self.log_dict(tqdm_dict, prog_bar=True)
+        output = OrderedDict({"loss": train_loss, "num_correct": num_correct})
 
         return output
 
     def training_epoch_end(self, outputs):
         """Compute and log training loss and accuracy at the epoch level."""
 
-        train_loss_mean = torch.stack([output['loss']
-                                       for output in outputs]).mean()
-        train_acc_mean = torch.stack([output['num_correct']
-                                      for output in outputs]).sum().float()
-        train_acc_mean /= (len(outputs) * self.batch_size)
-        return {'log': {'train_loss': train_loss_mean,
-                        'train_acc': train_acc_mean,
-                        'step': self.current_epoch}}
+        train_loss_mean = torch.stack([output["loss"] for output in outputs]).mean()
+        train_acc_mean = torch.stack([output["num_correct"] for output in outputs]).sum().float()
+        train_acc_mean /= len(outputs) * self.batch_size
+        self.log_dict({"train_loss": train_loss_mean, "train_acc": train_acc_mean, "step": self.current_epoch})
```

Comment:
maybe use Accuracy from the metrics package?

Reply:
This is a good idea, will go ahead and add/modify it.

Reply:
Done!